python_modules/dagster/dagster_tests/core_tests/graph_tests/test_op_selection.py

```diff
@@ -261,38 +250,37 @@ def _aliased():
     full_job = _aliased.to_job()
     result = full_job.execute_in_process()
     assert result.success
-    assert (
-        len(
-            [
-                evt.step_key
-                for evt in result.all_node_events
-                if evt.event_type == DagsterEventType.STEP_SUCCESS
-            ]
-        )
-        == 4
-    )
+    assert len(_success_step_keys(result)) == 4
 
     subsetted_job = _aliased.to_job(op_selection=["return_one_2*"])
     result_for_subset_def = subsetted_job.execute_in_process()
     assert result_for_subset_def.success
-    assert (
-        len(
-            [
-                evt.step_key
-                for evt in result_for_subset_def.all_node_events
-                if evt.event_type == DagsterEventType.STEP_SUCCESS
-            ]
-        )
-        == 2
-    )
+    assert len(_success_step_keys(result_for_subset_def)) == 2
 
     result_for_subset = subsetted_job.execute_in_process(op_selection=["return_one_2*"])
     assert result_for_subset.success
+    assert len(_success_step_keys(result_for_subset)) == 2
+
+
+def test_op_selection_on_implicit_alias():
+    @job
+    def _reuse_ops_job():
+        add_one(return_one())
+        add_one(return_one())
+        add_one(return_one())
+
+    result_1 = _reuse_ops_job.execute_in_process(op_selection=["return_one*"])
+
+    assert result_1.success
+    assert len(_success_step_keys(result_1)) == 2
```

Comment:
shouldn't this be 3? Or does the * imply that there's always more (doesn't include the empty char case)?

Reply:
it should be executing `return_one` and `add_one`. the remaining ops are: `["add_one_1", "return_one_1", "add_one_2", "return_one_2"]`